build/metadata.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# coding: utf-8
+"""
+Process manuscript metadata and author metadata
+"""
+import collections
+import pathlib
+
+import yaml
+
+
+def get_metadata_object(path):
+    """
+    Load all of the metadata from a yaml file, & return.
+
+    :param path: pathlib.Path to a metadata YAML file.
+    :return: structured metadata object.
+    """
+    class metadata(object):
```

Comment:
This class has to go. It's not pythonic and confusing.

If you really want immutability (which I don't think you do), you can look into a [`namedtuple`](https://docs.python.org/3/library/collections.html#collections.namedtuple).